test/tests/ver-1kb/ver-1kb.i

```diff
@@ -0,0 +1,240 @@
+nb_segments_TMAP7 = 20
+node_size_TMAP7 = '${units 1.25e-5 m}'
+long_total = '${fparse nb_segments_TMAP7 * node_size_TMAP7}' # m
+nb_segments_TMAP8 = 100
+simulation_time = '${units 10000 s}'
+temperature = '${units 500 K}'
+R = '${units 8.31446261815324 J/mol/K}' # ideal gas constant from PhysicalConstants.h
+initial_pressure_1 = '${units 1e5 Pa}'
+initial_pressure_2 = '${units 1e-10 Pa}'
+initial_concentration_1 = '${units ${fparse initial_pressure_1 / (R*temperature)} mol/m^3}'
+initial_concentration_2 = '${units ${fparse initial_pressure_2 / (R*temperature)} mol/m^3}'
+solubility = '${units ${fparse 1/(R*temperature)} mol/m^3/Pa}' # Henry's law solubility
+diffusivity = '${units ${fparse 4.31e-6 * exp(-2818/temperature)} m^2/s}'
+n_sorption = 1
+unit_scale = 1
+unit_scale_neighbor = 1
+
+[Mesh]
+  [generated]
+    type = GeneratedMeshGenerator
+    dim = 1
+    nx = ${nb_segments_TMAP8}
+    xmax = ${long_total}
+  []
+  [enclosure_1]
+    type = SubdomainBoundingBoxGenerator
+    input = generated
+    block_id = 1
+    bottom_left = '0 0 0'
+    top_right = '${fparse 1/3 * long_total} 0 0'
+  []
+  [enclosure_2]
+    type = SubdomainBoundingBoxGenerator
+    input = enclosure_1
+    block_id = 2
+    bottom_left = '${fparse 1/3 * long_total} 0 0'
+    top_right = '${fparse long_total} 0 0'
+  []
+  [breakmesh]
+    input = enclosure_2
+    type = BreakMeshByBlockGenerator
+    block_pairs = '1 2'
+    split_interface = true
+    add_interface_on_two_sides = true
+  []
```

Comment:
this approach actually does not enable distributed meshes. You currently get the following error:
```
The following error occurred in the MeshGenerator 'breakmesh' of type BreakMeshByBlockGenerator.
BreakMeshByBlockGenerator is not implemented for distributed meshes
```

I think we were using another approach previously, maybe revert to that. Or let me know if you are not sure how to do it. You should be able to find other examples in other cases.